src/inspector_socket_server.h

```diff
@@ -37,6 +38,10 @@ class SocketServerDelegate {
 
 class InspectorSocketServer {
  public:
+  static const std::string GetUrlProtocol() {
```

Comment:
The `const` qualifier doesn't do anything here, does it?  As well, this method doesn't seem to be used and doesn't strike me as super-useful either.